examples/libs/cardano-mesh/basic-tx/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "cardano-mesh-basic-tx",
+  "version": "1.0.0",
+  "main": "main.ts",
+  "dependencies": {
+    "@dfns/lib-cardano-mesh": "file:../../../packages/lib-cardano-mesh",
+    "@dfns/sdk": "file:../../../packages/sdk",
+    "@dfns/sdk-keysigner": "file:../../../packages/sdk-keysigner",
+    "@meshsdk/core": "1.5.10",
```

Comment:
add the @emurge dependency here.

Reply:
It is already included in meshsdk:

```
"node_modules/@meshsdk/core": {
      "version": "1.5.10",
      "license": "Apache-2.0",
      "dependencies": {
        "@emurgo/cardano-message-signing-browser": "1.0.1",
        "@emurgo/cardano-message-signing-nodejs": "1.0.1",
        "@emurgo/cardano-serialization-lib-browser": "11.5.0",
        "@emurgo/cardano-serialization-lib-nodejs": "11.5.0",
        "@emurgo/cip14-js": "3.0.1",
        "axios": "0.27.2",
        "bip39": "3.0.4",
        "nanoid": "3.3.4",
        "zod": "3.19.1"
      },
      "engines": {
        "node": ">=16"
      }
    },
```

But I will add it so we can chose the right version